apps/parse/lib/parse/transfers.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Parse.Transfers do
+  @moduledoc """
+  Parses `transfers.txt` CSV from GTFS zip
+
+      "from_stop_id","to_stop_id","transfer_type","min_transfer_time","min_walk_time","min_wheelchair_time","suggested_buffer_time","wheelchair_transfer"
+      "place-cntsq","1123","0","","","","","1"
+
+  """
+
+  use Parse.Simple
+
+  @doc """
+  Parses (non-header) row of `transfers.txt`
+
+  See `Model.Transfer.t` documentation for descriptions of the individual fields.
+  """
+  def parse_row(row) do
+    %Model.Transfer{
+      from_stop_id: copy(row["from_stop_id"]),
+      to_stop_id: copy(row["to_stop_id"]),
+      transfer_type: String.to_integer(row["transfer_type"]),
```

Comment:
This works for our feed, but per the documentation it's also possible for this to be empty. That would be the same as a 0.